nnpdf_data/nnpdf_data/commondata/NNPDF_POS_3P16GEV/kinematics_F2U.yaml

```diff
@@ -0,0 +1,161 @@
+bins:
+- x:
+    min: null
+    mid: 5.0e-07
+    max: null
+  Q2:
+    min: null
+    mid: 5.0
```

Comment:
```suggestion
    mid: 10.0
```
?

Reply:
In your own paper the recommended scale is 5GeV2

Reply:
yes, yes but look at the title of the PR

Reply:
> ?

Yes thanks, I had missed the kinematics, pinecard should be fine though